develop/parachain-devs/runtime-development/build-deterministic-runtime.md

```diff
@@ -0,0 +1,130 @@
+
+Over time, the tooling around the `srtool` Docker image has expanded to include the following tools and helper programs:
+
+- [`srtool-cli`](https://github.com/chevdor/srtool-cli){target=\_blank} provides a command-line interface to pull the srtool Docker image, get information about the image and tooling used to interact with it, and build the runtime using the `srtool` Docker container
+- [`subwasm`](https://github.com/chevdor/subwasm){target=\_blank} provides command-line options for working with the metadata and WebAssembly runtime built using srtool. The `subwasm` program is also used internally to perform tasks in the `srtool` image
+- [`srtool-actions`](https://github.com/chevdor/srtool-actions){target=\_blank} provides GitHub actions to integrate builds produced using the `srtool` image with your GitHub CI/CD pipelines
+- [`srtool-app`](https://gitlab.com/chevdor/srtool-app){target=\_blank} provides a simple graphical user interface for building the runtime using the `srtool` Docker image
+
+## Prepare the Environment
+
+To work with the Docker image that executes the code in the `srtool` Docker container, you must have a Docker account and Docker command-line or desktop tools available.
+
+It is recommended to install the `srtool-cli` program to work with the Docker image using a simple command-line interface.
+
+To prepare the environment:
+
+1. Open a terminal shell
+
+2. Verify that that Docker is installed by running the following command:
+
+   ```bash
+   docker --version
+   ```
+
+   If Docker is installed, the command displays version information:
+
+   ```text
+   Docker version 20.10.17, build 100c701
+   ```
+
+3. Install the `srtool` command-line interface by running the following command:
+
+   ```bash
+   cargo install --git https://github.com/chevdor/srtool-cli
+   ```
+
+4. View usage information for the `srtool` command-line interface by running the following command:
+
+   ```bash
+   srtool help
+   ```
+
+5. Download the latest `srtool` Docker image by running the following command:
+
+   ```bash
+   srtool pull
+   ```
+
+## Start a Deterministic Build
+
+After the environment has been prepared, the WebAssembly runtime can be compiled using the `srtool` Docker image.
+
+To build the runtime:
+
+```bash
+srtool build --app --package node-template-runtime --runtime-dir runtime
+```
+
+- The name specified for the `--package` should be the name defined in the `Cargo.toml` file for the runtime
+- The path specified for the `--runtime-dir` should be the path to the `Cargo.toml` file for the runtime
+- If the `Cargo.toml` file for the runtime is located in a `runtime` subdirectory, for example, `runtime/kusama`,  the `--runtime-dir` parameter can be omitted
+
+## Add Workflow Actions
+
+To add a GitHub workflow for building the runtime:
+
+1. Create a `.github/workflows` directory in the chain's directory
```

Comment:
It will be better to see the content of the files being added here. Also, I think using the steps of the GitHub UI to do this may not be the best way of explaining it. We can just say create the directory, and then add the file and show the `basic.yml` content in there keeping the link to the srtools-actions repo.